.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Current behavior
+<!-- Describe how the issue manifests. -->
+
+## Expected behavior
+<!-- Describe what the desired behavior would be. -->
+
```

Comment:
Maybe also add something like this?:

```
### Steps to Reproduce the Problem

### Location of a Minimal Repository that Demonstrates the Issue.
```

I'd say 90% of reported issues don't provide that and it means no-one can really look into them.

Reply:
Done :)